### PR TITLE
526 change logfile to debug_log.txt v4

### LINKED ISSUE #526: Change log.txt to debug_log.txt

### PR DESCRIPTION
**Pull request by mcdonaldcarolyn**

### CODE REVIEW

File: .github/workflows/test_tests.yml [hunk 1]
```diff
@@ -75,12 +75,12 @@ jobs:
         with:
           name: report
           path: ./**/*_report_*
-      - name: log artifact
+      - name: debug_log artifact
```
Comment (plocket) [inline .github/workflows/test_tests.yml]: A possible improvement could be changing the name to something like "upload debug_log". A bit more friendly to humans who don't know what "artifact" means in this context.

File: .gitignore [hunk 2]
```diff
@@ -128,4 +128,4 @@ _al_test_project_name*
 # So that our tests will behave more like our users' tests
 package-lock.json
 #report logs
-/log.txt
+debug_log.txt
```
Comment (plocket) [inline .gitignore]: We do still want the slash at the start.

**Review by plocket — APPROVED** [review]
Thank you for the improvement! I left two possible improvements that we can do in future if needed. If you want to do them now, let me know when you're done and I'll review again.

Also, if you could link this PR to issue #526, that would be useful.

Thank you!

File: .github/workflows/test_tests.yml (hunk 1 above)
Comment (mcdonaldcarolyn) [reply .github/workflows/test_tests.yml]: great idea!

File: .gitignore (hunk 2 above)
Comment (mcdonaldcarolyn) [reply .gitignore]: ok, got it. Thank you for the helpful feedback!

File: log.txt [hunk 3]
```diff
@@ -0,0 +1,984 @@
+Report:
```
Comment (plocket) [inline log.txt]: Just need to delete this file and then I think we're good!

**Review by plocket — CHANGES_REQUESTED** [review]
After the next comment is resolved I think we're good to go!

**Comment by plocket** [comment]
Oh, also link #526 if you have the time.

**Comment by mcdonaldcarolyn** [comment]
Closes #526

**Review by plocket — APPROVED** [review]
Looks great! Thanks for fine-tuning!

**Pull request merged**